runtime/src/lib.rs

```diff
@@ -118,20 +118,35 @@ impl modules::access::Config for Config {
     const METHOD_AUTHORIZATIONS: Lazy<modules::access::types::Authorization> = Lazy::new(|| {
         modules::access::types::Authorization::with_filtered_methods([(
             "evm.Create",
-            modules::access::types::MethodAuthorization::allow_from([
-                // ERC-721 factory contract.
-                Address::from_eth(
-                    H160::from_str("0xFdC4a5DEaCDfc6D82F66e894539461a269900E13")
-                        .unwrap()
-                        .as_ref(),
-                ),
-                // Main devnet deployment account.
-                Address::from_eth(
-                    H160::from_str("0x4B010D64C7b2037ea2Dabea4d303c4c24b723d00")
-                        .unwrap()
-                        .as_ref(),
-                ),
-            ]),
+            modules::access::types::MethodAuthorization::allow_from(
+                [
+                    // ERC-721 factory contract.
+                    Address::from_eth(
+                        H160::from_str("0xFdC4a5DEaCDfc6D82F66e894539461a269900E13")
+                            .unwrap()
+                            .as_ref(),
+                    ),
+                    // Main devnet deployment account.
+                    Address::from_eth(
+                        H160::from_str("0x4B010D64C7b2037ea2Dabea4d303c4c24b723d00")
+                            .unwrap()
+                            .as_ref(),
+                    ),
+                ]
+                .into_iter()
+                .chain(if is_devnet() {
```

Comment:
Is it useful the restrict this to devnet? We are also using the same key to deploy to testnet, and I expect that we will also use it for production later.

Reply:
That makes sense, yeah. I added the testnet as well.